src/stan/services/sample/fixed_param.hpp

```diff
@@ -80,6 +80,62 @@ int fixed_param(Model& model, const stan::io::var_context& init,
   return error_codes::OK;
 }
 
+template <typename Model, typename InitContextPtr, typename InitWriter,
+          typename SampleWriter, typename DiagnosticWriter>
+int fixed_param(Model& model, const std::size_t num_chains,
+                const std::vector<InitContextPtr>& init,
+                unsigned int random_seed, unsigned int chain,
+                double init_radius, int num_samples, int num_thin, int refresh,
+                callbacks::interrupt& interrupt, callbacks::logger& logger,
+                std::vector<InitWriter>& init_writer,
+                std::vector<SampleWriter>& sample_writers,
+                std::vector<DiagnosticWriter>& diagnostic_writers) {
+  std::vector<boost::ecuyer1988> rngs;
+  std::vector<Eigen::VectorXd> cont_vectors;
+  std::vector<util::mcmc_writer> writers;
```

Comment:
The other parallel versions all have a simple `if (num_chains == 1) { call_non_parallel_version() }` but this doesn't - is there a specific reason for that?

Reply:
Just forgot it, but added now